### DISCUSSION TITLE
Problem Displaying Fan Speed for Mill City MCR-10

### DISCUSSION THREAD

**Discussion by Contrabean** [artisan-roaster-scope/artisan#877] [Q&A]
I have two new MCR-10 Mill City roasters. I am having a problem displaying the fan speed in the extra LCD window. Readings should be from 0-30. On the config/port menu, for input 6 I have register as 4102 which is fan on the MCR. If I set the divider to 1/100, the LCD displays 00. If I set the diver to 1/10, the LCD displays 10x the Hz reading on the MCR control panel. 

![image](https://user-images.githubusercontent.com/28113349/166973355-624e3590-171a-4678-9ad6-a3ec7c49be94.png)

![image](https://user-images.githubusercontent.com/28113349/166973473-9409fefb-58bb-45bd-93cb-737a9457ce27.png)

![image](https://user-images.githubusercontent.com/28113349/166973520-30cad681-054b-48d7-9887-42a0b19805b7.png)

![image](https://user-images.githubusercontent.com/28113349/166973552-050b287d-0d89-45f0-8fa6-0ba3356e1ada.png)

Can anyone help me figure out what I am doing wrong?

Ron

**Comment by MAKOMO** [comment]
Ron, please attach your artisan settings .aset file renamed to .txt (export via menu Help >> Save Settings) as well as one .alog profile renamed to .txt that you recorded with those settings. We will take a look.

**Comment by Contrabean** [comment]
Marko,

Attached is what I hope you want. Also a PDF of a very busy graph (will likely eliminate some of the extraneous stuff moving forward.

Ron
________________________________

**Reply by poundy** [reply]
No Ron, that's not what we mean. Please upload the files to GitHub, not attach them in email. While it's somewhat possible to extract them I can't see the boundary between the two parts (the settings and the example roast profile).

**Comment by Contrabean** [comment]
Ok, I misunderstood.

Will do that tomorrow.

Ron

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Contrabean** [reply]
Uploaded as requested

**Comment by Contrabean** [comment]
[artisan-settings 7May2022.txt](https://github.com/artisan-roaster-scope/artisan/files/8645710/artisan-settings.7May2022.txt)
[Guate Washed 7May2022.txt](https://github.com/artisan-roaster-scope/artisan/files/8645712/Guate.Washed.7May2022.txt)

**Reply by MAKOMO** [reply]
Your fan, gas, drum speed data is most likely not communicated by the machine in Fahrenheit thus you have to choose the empty field for the unit marked in the screenshot below.

<img width="1160" alt="Screenshot 2022-10-11 at 10 48 51" src="https://user-images.githubusercontent.com/13909011/195044306-e4fb11d4-5b41-46b8-8382-83abea1d3275.png">

**Comment by Contrabean** [comment]
Thanks Marko.

Artisan is currently freezing whenever I try to use many of the menus. Can’t access config/curves or config/axes without artisan freezing.

Ron

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by MAKOMO** [reply]
Never happend to me. Must be a local problem on your computer. Just restart that machine.

**Comment by hernandemonteiro** [comment]


**Reply by poundy** [reply]
is your reply relevant to this discussion? What log are you talking about? Perhaps you will need to elaborate what you're wanting to achieve... and if not relevant to using Artisan on an MCR-10, I'd suggest you start a separate thread

**Reply by CursedSpiderBoi** [reply]
+1 that
